config/development.js

```diff
@@ -50,6 +50,19 @@ module.exports = {
       "-----END RSA PRIVATE KEY-----",
   },
 
+  botToken: {
+    publicKey:
```

Comment:
won't we need this in prod?

Reply:
We will be setting the keys from the hosting provider dashboard

Reply:
Yeah right